syft/formats/common/spdxhelpers/to_syft_model.go

```diff
@@ -352,6 +356,25 @@ func extractMetadata(p *spdx.Package2_2, info pkgInfo) (pkg.MetadataType, interf
 		return pkg.JavaMetadataType, pkg.JavaMetadata{
 			ArchiveDigests: digests,
 		}
+	case pkg.GoModulePkg:
+		var h1Digest string
```

Comment:
This should probably consolidated into a utility function somehow, but it's very specific to `h1` -> `sha256`